app/controllers/reviews_controller.rb

```diff
@@ -19,6 +19,12 @@ def edit
     @edit_review = Review.find(params[:id])
     @new_review = @edit_review
     @reviews = @podcast.reviews
+    @host_names = []
+    if !@podcast.hosts.empty?
+      @podcast.hosts.each do |host|
+        @host_names << host.name
+      end
+    end
     render :'/podcasts/show'
   end
```

Comment:
This controller action is getting a little long in the tooth. In a future pull request, you might consider ways to refactor it -- perhaps extracting some of the logic into a Plain Ol' Ruby Object?